README.md

```diff
@@ -79,6 +79,8 @@ Either way you choose to go you're gonna need a `pull-secret`... we are still in
 
 We've added a very simple `start.sh` script to make your life easier... if you want to deploy `OCM` the __"hard way"__ you can find the instructions for deploying `OCM` using nothing but `oc` commands [here](#manually-deploy-using-oc-commands-the-hard-way).
 
+0. First need to export KUBECONFIG=/path/to/some/cluster/kubeconfig
+
```

Comment:
The README already points out that `oc` and `kubectl` must be configured to connect to your target OCP cluster:  https://github.com/open-cluster-management/deploy#prereqs

Reply:
"oc & kubectl (ver. 1.16+) configured to connect to your OCP cluster"

we could make this a bit easier by showing the `export KUBECONFIG` and an `oc login`
either way, the script will fail right away if it doesn't make a cluster connection.